src/commands/node.mjs

```diff
@@ -1529,7 +1601,7 @@ export class NodeCommand extends BaseCommand {
         title: 'Load signing key certificate',
         task: async (ctx, task) => {
           const config = /** @type {NodeAddConfigClass} **/ ctx.config
-          const signingCertFile = Templates.renderGossipPemPublicKeyFile(constants.SIGNING_KEY_PREFIX, config.nodeId)
+          const signingCertFile = Templates.renderGossipPemPublicKeyFile(constants.SIGNING_KEY_PREFIX, ctx.newNode.name)
```

Comment:
We want to limit and possibly remove the use of storing objects directly on the ctx.  We wish to rewrite the way we are using config, but currently putting it in config will at least make it easier to migrate to a better solution in the future.